invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/landing_page/index.js

```diff
@@ -96,3 +97,41 @@ if (sidebarCommunitiesManageDiv) {
     sidebarCommunitiesManageDiv
   );
 }
+
+const requestAccessButton = document.getElementById("request-access-button");
+const handler = async () => {
+  // TODO i have a feeling this could be imported from somewhere else
+  const apiConfig = {
+    withCredentials: true,
+    xsrfCookieName: "csrftoken",
+    xsrfHeaderName: "X-CSRFToken",
+  };
+
+  const axiosWithConfig = axios.create(apiConfig);
+  const recid = requestAccessButton.dataset.recid;
+  const result = await axiosWithConfig.post(`/api/records/${recid}/access/request`, {
```

Comment:
is this a new endpoint? because we already have `api/records/<recid>/requests` reachable inside the record link, ie. record.links.requests, should we reuse that one ?

Reply:
yes, `POST /api/records/<recid>/access/requests` and `POST /api/records/<recid>/access/requests/guest` are the [new endpoints](https://github.com/inveniosoftware/invenio-rdm-records/pull/1306/files#diff-9a259f98a63719d79e15be76b47a7168338216f4c63b9150068bacb4972b1150R104-R105) for **creating** access requests.